www/src/crypto/murmur.js

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
Can you explain this whole file? This looks mainly like nonsense to me haha

Reply:
@abbeyhrt thankfully this is a file that is just copy-pasted around the internet lol. You can find the implementation for murmur hash on GitHub, but effectively it's a specific type of hash that is quick and generally useful for looking things up based on id.

This can be nice if you need something that generally needs a "stable" id, whether that's for a URL or for looking things up from different parts of a system. In terms of why murmur hash, it could 100% be replaced with any other hash function and we'd be fine 😂  it's just the one I've used most recently.